### PR TITLE
Search page, Search coin API

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added search page, and it's logic
*Added add to portfolio modal
https://deploy-preview-5--trade-book.netlify.app/search

### CODE REVIEW

**Review by omjadhav85 — APPROVED** [review]
Great work! Cool implementation of debounce logic in search to prevent unnecessary api calls. Also, loved how you have seperated out everything into individual components. Good to merge!

File: src/components/modals/modal.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.modal-wrapper {
+  position: absolute;
+  z-index: 1;
+  width: 100vw;
+  height: 100vh;
+  /* background-color: black; */
```
Comment (abhij1607) [inline src/components/modals/modal.css:6]: commented code could be removed

**Review by abhij1607 — APPROVED** [review]
Neat & Clean code
Good use of debouncing
Got an Idea on how to implement search feature

commented line in css could be removed

File: src/components/modals/modal.css (hunk 1 above)
Comment (AnuragSahu11) [reply src/components/modals/modal.css:6]: yes I will remove

**Pull request merged**